torchgeo/trainers/byol.py

```diff
@@ -308,66 +309,76 @@ def update_target(self) -> None:
 
 
 class BYOLTask(pl.LightningModule):
-    """Class for pre-training any PyTorch model using BYOL."""
+    """Class for pre-training any PyTorch model using BYOL.
+
+    Supports any available `Timm model
+    <https://rwightman.github.io/pytorch-image-models/>`_
+    as an architecture choice. To see a list of available pretrained
+    models, you can do:
+
+    .. code-block:: python
+
+        import timm
+        print(timm.list_models(pretrained=True)
```

Comment:
Should we remove the `pretrained=True` arg here? What's the default?

Reply:
The default is False. I think if you are starting of with a model, pretrained imagenet is most likely better than from scratch so more helpful to see those models.

Reply:
I would personally leave it off since all models are valid, not just pretrained models.